sources/packages/api/test/institution.e2e-spec.ts

```diff
@@ -0,0 +1,73 @@
+require("../env_setup");
+import { Test, TestingModule } from "@nestjs/testing";
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { AppModule } from "../src/app.module";
+import { KeycloakConfig } from "../src/auth/keycloakConfig";
+import { KeycloakService } from "../src/services/auth/keycloak/keycloak.service";
+
+describe("Users controller (e2e)", () => {
+  const clientId = "student";
+  // Nest application to be shared for all e2e tests
+  // that need execute a HTTP request.
+  let app: INestApplication;
+  // Token to be used for all e2e tests that need test
+  // the authentication endpoints.
+  // This token is retrieved from Keyclock.
+  let accesstoken: string;
+
+  beforeAll(async () => {
+    await KeycloakConfig.load();
+    const token = await KeycloakService.shared.getToken(
+      process.env.E2E_TEST_STUDENT_USERNAME,
+      process.env.E2E_TEST_STUDENT_PASSWORD,
+      clientId,
+    );
+    accesstoken = token.access_token;
+
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+    app = moduleFixture.createNestApplication();
+    await app.init();
+  });
+
+  //IN progress- Will unskip once there is a delete user in place to that this test can be run idempotently.
+  describe.skip("save institution", () => {
```

Comment:
Please remove the skip before commit.

Reply:
Discussed, Removed comment but left the skip because of BCeID calls